src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+* Introduced a new metric, `http.server.request.duration`, for users who opt
+  into the new semantic convention by configuring the `OTEL_SEMCONV_STABILITY_OPT_IN`
+  environment variable. This metric measures time in seconds and offers unique
+  histogram buckets as per the
+  [spec](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration)
+  , replacing the previous `http.server.duration metric`, which measured time in
+  milliseconds.
+  * Former buckets: `0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500,
```

Comment:
Would this qualify as a breaking change?

Reply:
This histogram is not a breaking change because it's only applied to the new metric.

Reply:
It's a new and a replacement metrics. If spec does not treat it as breaking change we are fine.

Reply:
Its not breaking change **now**, as it requires explicit opt-in.